### ISSUE TITLE
Add interface for viewing, adding and removing embedded cover art to tag editor

### ISSUE THREAD

**Issue by CreamyCookie** [quodlibet/quodlibet#3193]
The proposed change
-------------------
Add the ability to view, change and remove covert art of the currently selected album.

For example, EasyTag has this:

![screenshot from 2017-01-23 11-21-10](https://cloud.githubusercontent.com/assets/991986/22200144/2e9e7bda-e15e-11e6-9a07-764fb46a499b.png)

(screenshot from lazka)

Benefits of this change
-----------------------
Would make it easier for new users (no plugins to find and enable) and would be more efficient.

(Personally, I think embedding cover art is a bad idea, but I guess it can make sense for users for often transfer single files to another medium, for example using the playlist export plugin.)

**Comment by declension** [comment]
We already have the _Embedded Images_ plugin - do you want to add viewing to this?

**Comment by yochananmarqos** [comment]
@declension In a nutshell: We all want Mp3Tag with it's features on Linux. Puddletag was good, but development is almost dead and it needs a Python 3 + Qt5 refresh. That's the issue.

**Comment by CreamyCookie** [comment]
@declension Didn't see your comment until now. I moved the remaining part of #28 to this, so I'm not the original poster. That said, viewing the images with the information of file type + resolution + size would be nice.

**Comment by pearcemichal** [comment]
I agree that the functionality present in the "edit embedded images" plugin should just be added as a tab in the metadata editor, possibly in addition to a top level "paste cover image" menu option. 

I am currently working on getting started with contributing, so I will look into this at some point if no one else does.
